### ISSUE TITLE
Why are there no screenshots?

### ISSUE THREAD

**Issue by TechDufus** [jesseleite/GridLayout.spoon#1]
Seriously... why are there no screenshots / gifs showcasing this BEAST of a spoon???

**Comment by jesseleite** [comment]
BECAUSE IT'S NOT FOR THE MASSES, just for secret elite group of people who like being lua nerds

**Comment by jesseleite** [comment]
*I should record a lil video demo sometime!

**Comment by jesseleite** [comment]
Yeah, should do screenshots and a lil intro video, but let's flesh out multi-monitor support and squash some issues and use it for a while first. I'm in no rush for public adoption 😁

**Comment by TechDufus** [comment]
bump
